src/replication/DealReplicationService.ts

```diff
@@ -257,34 +282,39 @@ export default class DealReplicationService extends BaseService {
       const { outDir } = <GenerateCSVRequest>request.body;
```

Comment:
I think we should remove this API and add this functionality to CLI only. Basically CLI talks to database directly and constructs the csv file.